Add mobile popup question to msal-browser FAQ (#5431)
AzureAD/microsoft-authentication-library-for-js: lib/msal-browser/FAQ.md

Yes, we recommend the new [Powershell Graph SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell) for doing so. For instance, the script below creates an Azure AD application with redirect URI of type **SPA** and **User.Read** permission for Microsoft Graph in a tenant specified by the user, and then provisions a service principal in the same tenant based on this application object:

> ## Can I acquire tokens using popups on mobile browsers

Reviewer: Can we make this more general? i.e. "Does MSAL.js support mobile browsers?" and add a statement to the effect of "in general, yes ,however, given we do not test specifically test against mobile browsers support may be limited and some features and flows may not work. For problems specific to mobile browsers not listed here please open an issue with clear instructions to reproduce"

Reviewer: And then move this to be right below the browser support question

Reviewer 2: +1

Author: Updated, thanks for the feedback!

> While MSAL's `loginPopup` and `acquireTokenPopup` APIs should work on mobile browsers that support popups, using popup APIs is discouraged on mobile scenarios, mainly because not all browsers support popups. We recommend using redirect APIs such as `loginRedirect` and `acquireTokenRedirect` when your application is running in a mobile browser.

Reviewer: ```suggestion
While MSAL's `loginPopup` and `acquireTokenPopup` APIs may work on some mobile browsers, using popup APIs is discouraged on mobile devices as not all browsers will support this flow. We recommend using redirect APIs such as `loginRedirect` and `acquireTokenRedirect` when your application is running on a mobile device.
```
